optd-persistent/src/cost_model/catalog/mock_catalog.rs

```diff
@@ -110,23 +113,23 @@ impl MockCatalog {
             MockStatistic {
                 id: 1,
                 stat_type: StatType::Count as i32,
-                stat_value: "100".to_string(),
+                stat_value: json!(100),
                 attr_ids: vec![1],
```

Comment:
Does an attribute have the statistic type `Count`? IMO it should only be valid for table stats. Maybe change to cardinality?

Reply:
Oh I am not sure but I think `Cardinality` means the count of unique value? And attribute `Count` means not-null number of this column hmmm

Reply:
Not sure whether we should introduce sth like `NonNullCount` for this.

Reply:
From my perspective, I think `Count` is the not-null number hmmm, and this representation is clear compared to introducing `NonNullCount` since we can unify the table and attr stat types.

Reply:
If we use `Count` to represent both:
- record number when it's table stat
- non-null value number when it's attr stat
- 
tbh this is less clear to me than introducing `NonNullCount`... But I'm good for now. We can bring this up later.

Reply:
Done!